test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
+./test_scripts/Polices/External_UCS/002_ATF_N_TC_PreloadedPT_with_external_consent_status_groups_struct.lua Check_that_SDL_shut_down_in_case_external_consent_status_groups_struct_is_defined_in_PreloadedPT
+./test_scripts/Polices/External_UCS/003_ATF_P_TC_PreloadedPT_without_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_starts_successfully_in_case_disallowed_by_external_consent_entities_off_struct_is_omitted_in_PreloadedPT
+./test_scripts/Polices/External_UCS/004_ATF_P_TC_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_stores_disallowed_by_external_consent_entities_off_struct_in_LPT_in_case_it_is_defined_in_PreloadedPT
+./test_scripts/Polices/External_UCS/005_ATF_P_TC_PreloadedPT_without_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_starts_successfully_in_case_disallowed_by_external_consent_entities_on_struct_is_omitted_in_PreloadedPT
+./test_scripts/Polices/External_UCS/006_ATF_P_TC_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_stores_disallowed_by_external_consent_entities_on_struct_in_LPT_in_case_it_is_defined_in_PreloadedPT
+./test_scripts/Polices/External_UCS/007_ATF_N_TC_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua Check_that_SDL_shut_down_in_case_disallowed_by_external_consent_entities_off_struct_has_invalid_type_of_params_in_PreloadedPT
+./test_scripts/Polices/External_UCS/008_ATF_N_TC_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct_with_invalid_type_of_params.lua Check_that_SDL_shut_down_in_case_disallowed_by_external_consent_entities_on_struct_has_invalid_type_of_params_in_PreloadedPT
```

Comment:
@dboltovskyi, 
1. please specify commit with new scripts.
2. it's not enought clear for me. If you have structure:
“disallowed_by_external_consent_entities_on”: [{
"entityType": ,
"entityID": ,
}]
which of parameter will take "Value".
According to me you should have cases:
1: “disallowed_by_external_consent_entities_on”: [{
"entityType": Value,
"entityID": ,
}]
2: “disallowed_by_external_consent_entities_on”: [{
"entityType": ,
"entityID": Value,
}]
3. Some combination:
“disallowed_by_external_consent_entities_on”: [{
"entityType": Value / ,
"entityID": Value / ,
}]

May be additional case where
"disallowed_by_external_consent_entities_off”: [{
"entityType": ,
"entityID": ,
}]
is placed in other section(not "") in preloaded file.